src/averageAges.js

```diff
@@ -53,9 +70,23 @@ function calculateWomenAverageAge(people, withChildren) {
  * @param {boolean} onlyWithSon - optional
  *
  * @return {number}
+ *
+ *
+ *
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const childrens = people.filter(person =>
```

Comment:
Consider using more descriptive variable names, like `children` instead of `childrens`. 🤔